src/plugins/vis_type_timeseries/common/vis_schema.ts

```diff
@@ -180,6 +180,7 @@ export const seriesItems = schema.object({
   series_time_field: fieldObject,
   series_interval: stringOptionalNullable,
   series_drop_last_bucket: numberIntegerOptional,
+  series_ignore_daylight_time: numberIntegerOptional,
```

Comment:
are you sure you need it? I see that you set this param for panel options, not for series